JetUtilities/src/crystalBall.cc

```diff
@@ -0,0 +1,227 @@
+#include "JetMETAnalysis/JetUtilities/interface/crystalBall.h"
+
+#include "TMath.h"
+
```

Comment:
As a style I like to keep all of the headers in the header file of the class that uses them. It also prevents multiple linking issues.